Firestore/Source/API/FIRAggregateField+Internal.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#import "FIRAggregateField.h"
-#import "FIRFieldPath.h"
+#import "Firestore/Source/Public/FirebaseFirestore/FIRAggregateField.h"
```

Comment:
We need to better understand why this change is necessary

Reply:
I still do not understand it, but here are some observations:
1. `#import <FIRFieldPath.h>` works
2. Apart from the module name differences, I'm not seeing a difference in the compilation command when building this file via 10.16.0 and this branch
3. If I copy the failing command and paste it in terminal, the build error reproduces. Adding `-I/path/to/repo/Firestore/Source/Public` works.

Reply:
Discussed offline. Going to try integrating option 3 in the `FirebaseFirestoreInternal.podspec`'s xcconfig